docs/source/release_notes.rst

```diff
@@ -37,7 +37,7 @@ This collection can manage CICS and CICSPlex® SM resources and definitions by c
   * Sample playbooks are available at the `samples repository`_. Each playbook contains a README that explains what configurations must be made to run a sample playbook.
 
 .. _samples repository:
-   https://github.com/IBM/z_ansible_collections_samples/tree/master/cics/cmci
+   https://github.com/IBM/z_ansible_collections_samples
```

Comment:
Do we want to point to the samples _repository_, or to the CICS-specific _folder_ with our own README at https://github.com/IBM/z_ansible_collections_samples/tree/master/zos_subsystems/cics?

I like your proposed link because it allows you to see the whole context of all z/OS content, but I also like the idea of pointing at the CICS-specific folder to get our users to the right place quicker.

If we were to point to the CICS-specific README, I think we should add a link to the repository root in that README, so that we direct users straight to the CICS README but they become aware that there are other wider samples and concepts available too.

Reply:
For that last paragraph, I've raised IBM/z_ansible_collections_samples#49 (regardless of which way we decide to go here!).

Reply:
README of CICS root:

```suggestion
   https://github.com/IBM/z_ansible_collections_samples/tree/master/zos_subsystems/cics
```